### PR TITLE
Add Hindi Translation Support

### PR DESCRIPTION
**Pull request by KowshikaSinivasan**
**Add Hindi Translation Support**

Hindi is one of the most widely spoken languages in the world, with over 609.5 million speakers. It is the official language of India, a country with a population of over 1.4 billion people. Given its significance, it's essential to provide Hindi language support in digital platforms and applications to ensure accessibility for a large portion of the population.
 By supporting Hindi translations, we can make the platform more inclusive and user-friendly for millions of people, improving accessibility and engagement.

This pull request includes the Hindi translation, ensuring that Hindi-speaking users can interact with the content comfortably in their native language.
Files  or folders newly added:
      1.hi.yml   (_data/locales/hi.yml)
      2.hi folder with files index.html,maintainer.html and buissness.html

- [ yes] Have you followed the [contributing guidelines](https://github.com/github/opensourcefriday/blob/HEAD/CONTRIBUTING.md)?
- [yes] Have you explained what your changes do, and why they add value?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by KowshikaSinivasan** [comment]
Kindly only consider last commit.

**Comment by jmeridth** [comment]
@KowshikaSinivasan than you for your contribution.  `business.html` needs to be `businesses.html` please.  Rest looks good to me.

**Comment by KowshikaSinivasan** [comment]
I have made the necessary and updated the code .  

Since  this is my first contribution, I made this mistake. 
Thank you for your patience and guidance.

File: _data/locales/hi.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      title: "अपने वर्कफ़्लो को बेहतर बनाएं"
+      description: "अन्य मुख्य योगदानकर्ताओं से सीखें कि वे कैसे अधिक प्रभावी बनें।"
+      boundaries:
+        title: "सीमाएँ तय करें"
+        description: 'अपने प्रोजेक्ट के आकार और <a href="https://opensource.guide/best-practices/#its-okay-to-hit-pause">स्पष्ट सीमाएँ निर्धारित करें</a>। "नहीं" कहना उतना ही महत्वपूर्ण है जितना कि "हाँ" कहना।'
+      automation:
+        title: "स्वचालन"
+        description: '<a href="https://opensource.guide/best-practices/#bring-in-the-robots">बोट्स को लागू करें</a> ताकि आप परीक्षणों और अन्य सत्यापन कार्यों को अधिकतम स्वचालित कर सकें।'
+    visitor:
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (KowshikaSinivasan) [reply _data/locales/hi.yml]: I've made the necessary changes to resolve the merge conflicts. Please review and let me know if anything else needs attention. Thanks!

**Review by jmeridth — APPROVED** [review]

**Pull request merged**